content/support-matrix/delfin/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Delfin
+menuTitle: Delfin
+description: "Support matrix data for drivers of Delfin project"
+weight: 10
+disableToc: false
+tags: ["support matrix", "delfin", "drivers"] 
+---
+
+Delfin project aims to overcome a challenge to manage heterogeneous storage devices on a single platform.
+Delfin has standardized common API services. 
+
+Support matrix of each storage drivers are group to their vendor name.
```

Comment:
Support matrix of each storage drivers are group to their vendor name. -> _Support matrix of storage drivers are grouped by their vendors_

Reply:
done